lib/hako/schedulers/ecs_elb_v2.rb

```diff
@@ -158,6 +158,16 @@ def modify_attributes
                   elb_client.modify_listener(listener_arn: current_listener.listener_arn, ssl_policy: new_listener['ssl_policy'])
                 end
               end
+
+              if current_listener && new_listener['certificate_arn'] && new_listener['certificate_arn'] != current_listener.certificates[0]&.certificate_arn
+                certificates = [{ certificate_arn: new_listener['certificate_arn'] }]
+                if @dry_run
+                  Hako.logger.info("elb_client.modify_listener(listener_arn: #{current_listener.listener_arn}, certificates: #{certificates.inspect}) (dry-run)")
+                else
+                  Hako.logger.info("Updating ELBv2 listener #{new_listener['port']} certificates to #{certificates.inspect}")
+                  elb_client.modify_listener(listener_arn: current_listener.listener_arn, certificates: certificates)
```

Comment:
Do not call ModifyListener twice. They should be unified to one ModifyListener call.

Reply:
Fixed https://github.com/eagletmt/hako/pull/94/commits/bf07caf1f6cbe9b68ecbc6e9464d773aea302646